### PR TITLE
Learning Responsive Web Design

### PR DESCRIPTION
**Pull request by JuliCarracedo**
<h1>Learning Responisve Web Design</h1>
<h2>The project</h2>
<p>This is the second project on the curriculum. The goal is to introduce responsive web design to a website via the correct use of media queries on HTML5 and CSS3, using as a template <a target="_blank" href="https://perma.cc/M5ZV-Q2D6">this website</a>.</p>
<h2>What will we be doing?</h2>
We`ll be cloning the linked website in order to apply the same responsive design solutions the original programmer of the website applied.
<h2>Authors</h2>
<h3>Julian Carracedo</h3>
<p>Github: <a target="_blank" href="https://github.com/JuliCarracedo" >JuliCarracedo</a></p>
<h3>Yurvaj Manoo</h3>
<p>Github: <a target="_blank" href="https://github.com/YurvajM108" >YurvajM108</a></p>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        <div  class="topright-hl">
+             <h4 class="hl-title">Google unveils privacy-friendly AI cookie killer</h4>
+            <div class="hl-detail"><p>By Thomas Macauley </p><p>&#x1F553; 2 hours ago </p><p><img alt="share" src="resources/iconmonstr-share-1-16.png"> 4</p></div>
+        </div>
+
+        <div  class="bottomright-hl">
+            <h4 class="hl-title">This Mac app automatically detects your duplicate files</h4>
+            <div class="hl-detail"><p>By Ivan Metha </p><p>&#x1F553; 2 hours ago </p><p><img alt="share" src="resources/iconmonstr-share-1-16.png"> 5</p></div>
+        </div>
+    </div>
```
Comment (jcy2704) [inline index.html]: - **Try to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)**

**For example, in this case, we can use the `<section>` and `<article>` tag for these and other articles.**

```
  <section class="highlights">
    <article class="left-hl">
      ...
    </article>
  </section>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class="navbar">
```
Comment (jcy2704) [inline index.html]: - **Try to follow the [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)**

- [ ] **Enclose the navbar inside a `<header>` tag**

File: index.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+                    <a class="black-nav-link" target="_self" href="#">Avertise</a>
+                    <a class="black-nav-link" target="_self" href="#">Jobs</a>
+                    <a class="black-nav-link" target="_self" href="#">Contact</a>
+                </div>
+
+            </div>
+            <div class="invisible-black-menu">
+                <div class="invisible-logo">
+                    <img alt="logo" class="inv-logo" src="https://noknok.com/wp-content/uploads/2019/10/tnw-logo-white.png">
+                </div>
```
Comment (jcy2704) [inline index.html]: - [ ] **The logo needs to be enclosed in an `<a>` tag.**

File: index.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+                    <a class="white-nav-link" target="_self" href="#">SHIFT</a>
+                    <a class="white-nav-link" target="_self" href="#">NEURAL</a>
+                </div>
+
+                <div class="search-div">
+                    <img class="search-icon" alt="search" src="resources/iconmonstr-magnifier-5-240.png">
+                </div>
+                <div class="hamburger-button">
+                    <img class="hb-icon" alt="hb-icon" src="resources/hamburger-menu.png">
+                </div>
```
Comment (jcy2704) [inline index.html]: - [ ] **These two icons need to be enclosed in an `<a>` tag for each.**

**Review by jcy2704 — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED!!😗

## Hey @JuliCarracedo @YuvrajM108!! Good job trying to clone TheNextWeb. 👍

### I know this project is kind of hard at the beginning, but your project is a good beginning. 🙌 However, there are many things we need to improve so your project can be approved. But no worries, as a TSE, it's my work to help you achieve it.

### First, let's talk about your README 📝. 
- **Your README looks good and it's really descriptive, I like it! 😌 But there are things you are missing. 😕**

  - [ ] You need to add instructions for new users to clone your project so they can use it on their own computer. You can use this [template](https://github.com/microverseinc/readme-template) to make your README look even better.

  - [ ] Fix all the changes required and change all screenshots.
 
### **Now, let's talk about your project 📁.**

- **As I said is a good beginning, but let's see what we need to improve. 🧐**

  - **PLEASE use the link offered to guide yourself [TNW](https://perma.cc/M5ZV-Q2D6) and try to make it similar to the original one.**

- Original
image

- Clone
image

   - **If we compare both images, we see that the 3 main articles have too much whitespace compared to the original.**
   - **It also seems that there are extra margins or padding on the sides when there is none on the original.**

- Original
image

- Clone
image

   - **Comparing both images, the original has more space on the sides compared to your project.**
   - **The items are not aligned correctly as you can see.**

### **FOR MORE CHANGES SEE BELOW 👇**

<hr>

### Keep the awesome work! 💪 Happy Coding for y'all!! 💻

**If you have any questions you can ask me on my Slack.**

**Best,**

**[💁‍♂️ Steven Jack Chung Yau](https://stevenchung.me)**

[LinkedIn](https://linkedin.com/in/stevenjchung) | [Twitter](https://twitter.com/_yiak) | [GitHub](https://github.com/jcy2704)

**Review by smcommits — APPROVED** [review]
# Status: Approved :+1: 
## Hello, @JuliCarracedo and @YuvrajM108! You guys have done an amazing job. The website appears to be great. Good work addressing the issues pointed out by the previous TSE. This PR is now approved. Congratulations and good luck in the future. 

## Optional Suggestions 
#### Please note that the following suggestions are optional 

- [ ] **The width of the navbar is not responsive, you should try to fix it** 

incimpnav

- [ ]  **The TNW logo is not appropriately responsive and overlaps other elements. You can fix this**
logoNOt

- [ ] **You should make use of more semantic HTML tags like `section` and `article`. It immensely helps with SEO.**

**Pull request merged**